setup.py

```diff
@@ -1,11 +1,21 @@
-from setuptools import setup
+import setuptools
+with open("README.md", "r") as fh:
+    long_description = fh.read()
 
-setup(name='ir_webstats_rc',
+setuptools.setup(
+      name='ir_webstats_rc',
       version='1.3.2',
-      description='A version of ir_webstats hacked on by Rob Crouch',
-      url='http://github.com/fuzzwah/ir_webstats_rc',
       author='Rob Crouch',
       author_email='rob.crouch@gmail.com',
-      license='MIT',
-      packages=['ir_webstats_rc'],
+      description='A version of ir_webstats hacked on by Rob Crouch',
+      long_description=long_description,
+      long_description_content_type="text/markdown",
+      url='http://github.com/fuzzwah/ir_webstats_rc',
+      packages=setuptools.find_packages(),
```

Comment:
Updated package detection, but did not remove relative importing as stated in commit message